docs/source/API/containers/DualView.rst

```diff
@@ -152,11 +152,11 @@ Description
        The arguments that follow are the dimensions of the View objects. For example, if the View has three dimensions,
        the first three integer arguments will be nonzero, and you may omit the integer arguments that follow.
 
-    .. cppkokkos:function:: DualView(const Impl::ViewCtorProp<P...>& arg_prop, typename std::enable_if<!Impl::ViewCtorProp<P...>::has_pointer, size_t>::type const n0 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n1 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n2 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n3 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n4 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n5 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n6 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n7 = KOKKOS_IMPL_CTOR_DEFAULT_ARG);
+    .. cppkokkos:function:: DualView(/* is-alloc-prop */ const& arg_prop, const size_t n0 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n1 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n2 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n3 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n4 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n5 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n6 = KOKKOS_IMPL_CTOR_DEFAULT_ARG, const size_t n7 = KOKKOS_IMPL_CTOR_DEFAULT_ARG);
 
        Constructor that allocates View objects on both host and device. This constructor works like the analogous constructor of View.
```

Comment:
Instead of "This constructor works like the analogous constructor of View," would something along the lines of "The constructor for DualView is similar to that of View" be a bit  clearer and true to the intended meaning?